src/main/c/Objects/pyjtype.c

```diff
@@ -152,13 +180,130 @@ static PyObject* getBaseTypes(JNIEnv *env, PyObject *fqnToPyType, jclass clazz)
     return bases;
 }
 
+
+
+/*
+ * Adding empty slots to the type dict to prevent creation of __dict__ for
+ * every instance and ensures all attributes go through pyjfield.
```

Comment:
Methods don't go through pyjfield, right?  I don't think this comment is right when it says "all attributes go through pyjfield".

Reply:
Yes, I have updated the comment. I was looking at it only from the perspective of assigning values to attributes since that was a problem I was seeing but it is more clear to mention all the pyjthings.